src/linter/messages.ts

```diff
@@ -333,6 +334,15 @@ export const MESSAGE_INFO = {
 		details: () => `"{@link sap.ui.core.RenderManager#methods/icon RenderManager}",`,
 	},
 
+	[MESSAGE.NOT_STATIC_CONTROL_RENDERER]: {
+		severity: LintMessageSeverity.Warning,
+		ruleId: RULES["no-deprecated-api"],
```

Comment:
I wouldn't expect this an `deprecated API`, as it never was intended to be defined that way. Therefore I think we should use a different ruleId. Also, it is not related to UI5 2.x, but a general check.

Reply:
True, thanks!
But, to me this one seems too specific. Would `static-var-required` be ok, then?

Reply:
Maybe `ui5-class-declaration`, as it is about how UI5 classes are declared/defined.
What do you think @codeworrior / @flovogt / @RandomByte ?

Reply:
just another idea `ui5-control-definition`?

Reply:
As the `renderer` is not a general concept for UI5 classes, I rather tend to `ui5-control-declaration`. But there might not be much in addition to this check that would justify a separate rule-ID for that. So I would be fine with `ui5-class-declaration`, too.

Reply:
Changed it to `ui5-class-declaration`. Let me know if something else comes